common/on-chip-calib.cpp

```diff
@@ -862,24 +956,31 @@ namespace rs2
                   ",\n \"step count\":" << step_count <<
                   ",\n \"apply preset\":" << (apply_preset ? 1 : 0) <<
                   ",\n \"accuracy\":" << accuracy <<
-                  ",\n \"scan only\":" << 0 <<
+                  ",\n \"scan only\":" << (host_assistance ? 1 : 0) <<
                   ",\n \"interactive scan\":" << 0 <<
                   ",\n \"depth\":" << 0 << "}";
         }
         std::string json = ss.str();
 
-        auto invoke = [](std::function<void()>) {};
-        int frame_fetch_timeout_ms = 3000;
-        rs2::depth_frame f = fetch_depth_frame(invoke, frame_fetch_timeout_ms);
-        rs2_metadata_type frame_counter = 0;
-        _progress = 0;
-
+        float health[2] = { -1.0f, -1.0f };
         auto calib_dev = _dev.as<auto_calibrated_device>();
         if (action == RS2_CALIB_ACTION_TARE_CALIB)
-            _new_calib = calib_dev.run_tare_calibration(ground_truth, json, [&](const float progress) {_progress = progress;}, 5000);
+            _new_calib = calib_dev.run_tare_calibration(ground_truth, json, health, [&](const float progress) {_progress = progress; }, 5000);
```

Comment:
This may be a new API but that's because this is a modified algorithm that now returns "health". There is no longer an API without the "health" parameter so there is no need for a different function name.